tests/bal_business_logic_suite.py

```diff
@@ -367,6 +373,24 @@ def test_when_hasCapability_called_on_managerInterface_then_has_mandatory_capabi
         )
 
 
+class Test_entityTraits(FixtureAugmentedTestCase):
+    def test_when_missing_entity_queried_for_write_then_empty_trait_set_returned(self):
```

Comment:
[Consider] I wonder if the fact that BAL returns all the traits it knows about for write is a business logic testable thing. Like check that kWrite + kRead are the same traitsets, minus the implicit versionTrait.